can/io/mf4.py

```diff
@@ -0,0 +1,392 @@
+"""
```

Comment:
Should this class also be added to the `lookup` dictionaries in the `logger.py` and `player.py` files?

Reply:
I've added them